metadata-ingestion/src/datahub/emitter/mce_builder.py

```diff
@@ -325,9 +336,9 @@ def make_dashboard_urn(
 ) -> str:
     # FIXME: dashboards don't currently include data platform urn prefixes.
     if platform_instance:
-        return f"urn:li:dashboard:({platform},{platform_instance}.{name})"
+        return f"urn:li:dashboard:({UrnEncoder.encode_string(platform)},{UrnEncoder.encode_string(platform_instance)}.{UrnEncoder.encode_string(name)})"
```

Comment:
Can we just migrate everything to use the typed urn classes?

Reply:
we can do something like this in the interim

```suggestion
        return DashboardUrn(platform, f"{platform_instance}.{name})"
```

It's also pretty easy to add those methods - see https://github.com/datahub-project/datahub/blob/12eb0cd1a7dbbd53f7a994a5502f52e66607cd7c/metadata-ingestion/scripts/avro_codegen.py#L429

Reply:
addressed in dab0d75f838a953cd7650ca566fe02d8fb2d7594